games-emulation/es-de/es_de-3.1.0.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="Gaming frontend for your multi-platform collection"
+DESCRIPTION="ES-DE (EmulationStation Desktop Edition) is a frontend for browsing and launching games from your \
+multi-platform collection. It comes preconfigured for use with a large selection \
+of emulators and game engines."
+HOMEPAGE="https://es-de.org"
+COPYRIGHT="2024 Northwestern Software AB"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="842c9966eb73efb3436a4df2dfdd66063ce7361c"
+SOURCE_URI="https://gitlab.com/es-de/emulationstation-de/-/archive/$srcGitRev/emulationstation-de-$srcGitRev.tar.gz"
+CHECKSUM_SHA256="52c820beddba7e08014f589120c087d6b84b79a88b6213bf13a4e025fd728285"
+SOURCE_FILENAME="emulationstation-de-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="emulationstation-de-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+        es_de = $portVersion
+        app:es_de = $portVersion
+        "
+
+REQUIRES="
+        haiku$secondaryArchSuffix
+        lib:libGL$secondaryArchSuffix
+        lib:libavcodec$secondaryArchSuffix
+        lib:libavformat$secondaryArchSuffix
+        lib:libavutil$secondaryArchSuffix
+        lib:libcurl$secondaryArchSuffix
+        lib:libfreeimage$secondaryArchSuffix
+        lib:libfreetype$secondaryArchSuffix
+        lib:libharfbuzz$secondaryArchSuffix
+        lib:libgit2$secondaryArchSuffix
+        lib:libicuuc$secondaryArchSuffix
+        lib:libintl$secondaryArchSuffix
+        lib:libpoppler$secondaryArchSuffix
+        lib:libpugixml$secondaryArchSuffix
+        lib:libSDL2_2.0$secondaryArchSuffix
+        "
+
+BUILD_REQUIRES="
+        haiku${secondaryArchSuffix}_devel
+        devel:libGL$secondaryArchSuffix
+        devel:libavcodec$secondaryArchSuffix
+        devel:libavformat$secondaryArchSuffix
+        devel:libavutil$secondaryArchSuffix
+        devel:libcurl$secondaryArchSuffix
+        devel:libfreeimage$secondaryArchSuffix
+        devel:libfreetype$secondaryArchSuffix
+        devel:libharfbuzz$secondaryArchSuffix
+        devel:libgit2$secondaryArchSuffix
+        devel:libicuuc$secondaryArchSuffix
+        devel:libintl$secondaryArchSuffix
+        devel:libpoppler$secondaryArchSuffix
+        devel:libpugixml$secondaryArchSuffix
+        devel:libSDL2_2.0$secondaryArchSuffix
+        "
+
+BUILD_PREREQUIRES="
+        cmd:g++$secondaryArchSuffix
+        cmd:ld$secondaryArchSuffix
+        cmd:cmake$secondaryArchSuffix
+        cmd:make
+        cmd:msgfmt$secondaryArchSuffix
+        cmd:pkg_config$secondaryArchSuffix
```

Comment:
Please sort these alphabetically.

Also, for cmake, there is no $secondaryArchSuffix.

Reply:
Hi, I have made these changes now!

Reply:
Actually reading this again I'm not sure if I perhaps misunderstood, do you mean that I should have removed $secondaryArchSuffix rather than add it to make?

Reply:
Never mind, I looked at some other recipes and I've now updated the ES-DE recipe accordingly!